packages/fxa-content-server/app/tests/spec/views/pair/auth_allow.js

```diff
@@ -56,6 +56,9 @@ describe('views/pair/auth_allow', () => {
       window: windowMock,
     });
     broker.set('remoteMetaData', REMOTE_METADATA);
+    broker.set('browserSignedInAccount', {
```

Comment:
You could probably add some asserts below that email address gets rendered in correct place.